CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-rc.5 July 28, 2021
+
+**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+
+## Breaking changes ⚠️
+* Improve camera API consumption from java programming language by adding a JvmName annotations. ([#495](https://github.com/mapbox/mapbox-maps-android/pull/495))
+* Rename AttributionView#setOnClickListener to setViewOnClickListener to avoid overloading the Android SDK method. Results in compilation on Android P and above. Adjust codebase to changes in enforced nullability of Android SDK code. ([#497](https://github.com/mapbox/mapbox-maps-android/pull/497))
+
+## Features ✨ and improvements 🏁
+* Add showLogo and showAttributes config for snapshotter. User can then hide logo and attributions in a snapshotter ([#496](https://github.com/mapbox/mapbox-maps-android/pull/496))
+* Add lifecycle plugin, users don't need to call onStart/onStop/onDestroy methods with this plugin ([#485](https://github.com/mapbox/mapbox-maps-android/pull/485))
```

Comment:
```suggestion
* Add lifecycle plugin so there is no need to call onStart/onStop/onDestroy methods explicitly ([#485](https://github.com/mapbox/mapbox-maps-android/pull/485))
```

Reply:
I guess we should maybe mention the limitations as well, wdyt?